src/osdag/design_type/compression_member/Column.py

```diff
@@ -573,6 +573,18 @@ def func_for_validation(self, design_dictionary):
         if flag:
             print(f"\n design_dictionary{design_dictionary}")
             self.set_input_values(self, design_dictionary)
+            if self.design_status ==False and len(self.failed_design_dict)>0:
+                logger.error(
+                    "Design Failed, Check Design Report"
+                )
+                return # ['Design Failed, Check Design Report'] @TODO
+            elif self.design_status:
+                pass
+            else:
+                logger.error(
+                    "Design Failed. Slender Sections Selected"
```

Comment:
Keep error consistent. There is a `,` first and `.` in the next. Also, if existing errors end in `.` use that here as well.

Reply:
These are the codes, that have been pasted from the flexure files, and so I left it as it is they are in that file.